web/components/FcDrawerViewData.vue

```diff
@@ -102,6 +102,19 @@
             :count-summary="countSummary"
             :loading="loadingCounts"
             @show-reports="actionShowReports" />
+          <div class="pa-5">
+            <div
+              v-for="study in studiesPending"
+              :key="study.id"
+              class="align-center d-flex">
+              <v-icon
+                color="warning"
+                left>mdi-information</v-icon>
+              <div>
+                {{study.studyType}} has been requested on {{study.createdAt | date}}.
```

Comment:
show the human-readable name.  (this can wait until #309 , since that should make this change easier.)